src/index.html

```diff
@@ -13,6 +13,136 @@
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--4">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--5">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
   </body>
```

Comment:
The content inside the <body> tag should be indented with 2 spaces for consistency with the HTML formatting guidelines.